WIP-CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   If you have the nfs-provisioner or the local-pv-provisioner installed, they will be left untouched when upgrading.
   You are responsible for managing them and/or removing them, and any unsused storageClasses.
 
+- InfluxDB is deprecated and Thanos is now enabled by default.
+   - With the removal of InfluxDB, the backups and buckets can eventually be removed.
```

Comment:
```suggestion
  If you have the nfs-provisioner or the local-pv-provisioner installed, they will be left untouched when upgrading.
  You are responsible for managing them and/or removing them, and any unsused storageClasses.
- InfluxDB is removed and Thanos is now enabled by default.
   - With the removal of InfluxDB, the backups and buckets can eventually be removed.
 - Running without object storage is no longer supported since it is required for Thanos.
   - The dev flavor is now updated to use s3 by default   
```